README.md

```diff
@@ -72,27 +74,31 @@ cannot be used as-is for any kind of geo-coding or validation
 |XX    |   210| 0.0|                                                                   |
 
 
-`Zip`: all non-missing values are in valid 5 digit format, but not all 
+#### Zip
+
+All non-missing values are in valid 5 digit format, but not all 
 of those match to real zip codes. Further validation pending. Note also 
 that just because a zip code is valid does not mean it can be mapped to 
 a ZCAT (e.g., PO Box Zips)
 
+#### NonProfit
 
-For `RaceEthnicity`, `Gender`, and `Veteran`, most of the data are 
-"Not Answered" due to these questions being optional.
-
+Has only Y or NA values, and so can be assumed to be a required question, 
+implying actual Missingness of 0%
 
-`NonProfit` has only Y or NA values, and so can be assumed to be a required
-question, implying actual Missingness of 0%
+#### JobsRetained
 
-
-`JobsRetained` contains some improbable values, and many values are Zero:
+contains some improbable values, and many values are Zero:
```

Comment:
in the effort to continue to test functionality: suggest this "contains" be capitalized to "Contains".

Reply:
looks good to me!